helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -412,17 +411,36 @@ private StoppableCheck performCustomInstanceCheck(String clusterId, String insta
     return instanceStoppableChecks;
   }
 
-  private Map<String, String> getCustomPayLoads(String jsonContent) throws IOException {
+  public static Map<String, String> getMapFromJsonPayload(String jsonContent) throws IOException {
     Map<String, String> result = new HashMap<>();
     if (jsonContent == null) {
       return result;
     }
+
     JsonNode jsonNode = OBJECT_MAPPER.readTree(jsonContent);
     // parsing the inputs as string key value pairs
     jsonNode.fields().forEachRemaining(kv -> result.put(kv.getKey(), kv.getValue().asText()));
     return result;
   }
 
+  public static List<String> getListFromJsonPayload(JsonNode jsonContent) throws IOException {
+    if (jsonContent == null) {
+      return new ArrayList<>();
```

Comment:
`Collections.emptyList()` might be better here?

Reply:
TFTR. Updated.